data.sql

```diff
@@ -1,5 +1,11 @@
-/* Populate database with sample data. */
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
```

Comment:
A possible code review comment is:

Hi, thank you for your submission. I have reviewed your code and found some errors and areas for improvement. Here are some suggestions:

- Please do not insert the id column values manually, as they are supposed to be autoincremented by the database. This is a requirement for this project and you need to make this change. You can omit the id column from the INSERT statements and let the database generate the values automatically.
- [OPTIONAL] Please use the same date format as specified in the instructions, which is YYYY/MM/DD. You can use slashes (/) instead of hyphens (-) to separate the year, month and day values. This will make your code more consistent and readable.
- [OPTIONAL] Please use consistent decimal places for the weight_kg column values, as specified in the instructions. You can use one decimal place for all the values, such as 8.0 instead of 8. This will make your data more accurate and uniform.

I have provided a corrected version of your code below. Please review it and make the necessary changes. If you have any questions or doubts, please let me know.

```.sql
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
VALUES ('Agumon', '2020/02/03', 0, true, 10.2);

INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
VALUES ('Gabumon', '2018/11/15', 2, true, 8.0);

INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
VALUES ('Pikachu', '2021/01/07', 1, false, 15.0);

INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
VALUES ('Devimon', '2017/05/12', 5, true, 11.0);

OR // Using of one insert statement with multiple values

INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (‘Agumon’, ‘2020/02/03’, 0, true, 10.2), (‘Gabumon’, ‘2018/11/15’, 2, true, 8.0), (‘Pikachu’, ‘2021/01/07’, 1, false, 15.0), (‘Devimon’, ‘2017/05/12’, 5, true, 11.0);
```